lua/entities/gmod_tardis/modules/sh_time_distortions.lua

```diff
@@ -1,12 +1,21 @@
 -- Support for time distortion generator by parar020100 and JEREDEK
 
-local search_radius = 1000
+local search_radius = 2048
 local explode_radius = 300
 
 local function TimeDistortionsPresent(pos, radius)
     for i,v in ipairs(ents.FindInSphere(pos, radius)) do
-        if v:GetClass() == "gmod_time_distortion_generator" and v:GetEnabled() then
-            return true
+        if(v:GetClass() == "gmod_time_distortion_generator" and v.On) then
```

Comment:
Unless you have a good reason I think we can leave this to be `v:GetEnabled()` as we ideally shouldn't be directly accessing properties on other entities, same for the Radius should probably have a `GetRadius` function

Reply:
I don't have a good reason, I accidentally'd the GetEnabled and only noticed right before commit

Reply:
Fixed